### PR TITLE
Update all.json

### PR DESCRIPTION
**Pull request by dubstard**
"app.uniswap.org.v3-withdraw.quest",
"coinbase.com-accountsecure01.com",
"coinbase.com-accountsecure02.com",
"coinbase.com-accountsecure03.com",
"coinbase.com-accountsecure04.com",
"coinbase.com-accountsecure05.com",
"connectwalletnodes.com",
"correctwservice.com",
"dappprotocolonline.online",
"dappsonlineconnect.com",
"erc20rescue.tools",
"finessejobs.info",
"fixedcoinwallets.com",
"improvewalletchains.com",
"klimadaov2.finance",
"mainnetdaps.com",
"manualwalletrestore.com",
"metamask-app.site",
"metamask-supportdesk.com",
"metamask-synch.com",
"metamask.is",
"metamask.recoverywallet.me",
"metamaskdownload.io",
"metamaskvip.com",
"metamaskvip.net",
"mintsync.org",
"mobwallet.co",
"multibridggedapps.com",
"recoverywallet.me",
"restorationwallet.com",
"tokenextensions.org",
"v3-uniswap-token.im",
"v3-withdraw.quest",

you know it's fake when the URL says "secure", haha

killing multiple birds with one trap
image

### CODE REVIEW

File: all.json [hunk 1]
```diff
@@ -169,6 +170,11 @@
     "coinbase-verifymai.servehttp.com",
     "coinbase-wallet.online",
     "coinbase-wesupports.servequake.com",
+    "coinbase.com-accountsecure01.com",
+    "coinbase.com-accountsecure02.com",
+    "coinbase.com-accountsecure03.com",
+    "coinbase.com-accountsecure04.com",
+    "coinbase.com-accountsecure05.com",
```
Comment (jacogr) [inline all.json:177]:
```suggestion
    "coinbase.com-accountsecure05.com",
    "com-accountsecure01.com",
    "com-accountsecure02.com",
    "com-accountsecure03.com",
    "com-accountsecure04.com",
    "com-accountsecure05.com",
```

File: all.json [hunk 2]
```diff
@@ -768,16 +788,19 @@
     "metamask.hellokidsbd.com",
     "metamask.io-updated.app",
     "metamask.io-wallet.me",
+    "metamask.is",
     "metamask.lav09dev.tk",
     "metamask.metmas-k.co",
     "metamask.org.in",
     "metamask.plantsvundead.com",
+    "metamask.recoverywallet.me",
```
Comment (jacogr) [inline all.json:796]:
```suggestion
    "metamask.recoverywallet.me",
    "recoverywallet.me",
```

**Review by jacogr — APPROVED** [review]

**Comment by dubstard** [comment]
Thanks for adding the main domains, I did it for a few of the FQDNs, but I missed some.
Thanks and cheers!

**Comment by jacogr** [comment]
No issues, it was quick.

**Pull request merged**